auth-web/src/components/auth/manage-business/AffiliatedEntityTable.vue

```diff
@@ -248,159 +260,188 @@ export default class AffiliatedEntityTable extends Mixins(DateMixin) {
     return items
   }
 
-  /** Returns true if the affiliation is a Name Request */
-  private isNameRequest (corpType: string): boolean {
-    return corpType === CorpType.NAME_REQUEST
+  /** Returns true if the affiliation is a Name Request. */
+  protected isNameRequest (business: Business): boolean {
+    return (business.corpType.code === CorpTypes.NAME_REQUEST && !!business.nameRequest)
   }
 
-  /** Returns true if the affiliation is a Temp Registration */
-  private isTemporaryBusinessRegistration (corpType: string): boolean {
-    return corpType === CorpType.NEW_BUSINESS || corpType === CorpType.NEW_REGISTRATION
+  /** Returns true if the affiliation is a temporary business. */
+  protected isTemporaryBusiness (business: Business): boolean {
+    return (
+      business.corpType.code === CorpTypes.INCORPORATION_APPLICATION ||
+      business.corpType.code === CorpTypes.REGISTRATION
+    )
   }
 
-  /** Returns true if the affiliation is a Numbered Company */
+  /** Returns true if the affiliation is a numbered IA. */
   private isNumberedIncorporationApplication (item: Business): boolean {
-    return item.corpType.code === CorpType.NEW_BUSINESS && item.name === item.businessIdentifier
+    return (
+      item.corpType.code === CorpTypes.INCORPORATION_APPLICATION &&
+      item.name === item.businessIdentifier
+    )
   }
 
-  /** Returns true if the affiliation is approved to start an IA or Registration */
-  private canUseNameRequest (business: Business): boolean {
+  /** Returns true if the affiliation is approved to start an IA or Registration. */
+  protected canUseNameRequest (business: Business): boolean {
     // Split string tokens into an array to avoid false string matching
     const supportedEntityFlags = LaunchDarklyService.getFlag(LDFlags.IaSupportedEntities)?.split(' ') || []
 
-    return this.isNameRequest(business.corpType.code) && // Is this a Name Request
-      business.nameRequest?.enableIncorporation && // Is the Nr state approved (conditionally) or registration
-      supportedEntityFlags.includes(business.nameRequest?.legalType) // Feature flagged Nr types
+    return (
+      this.isNameRequest(business) && // Is this a Name Request
+      business.nameRequest.enableIncorporation && // Is the Nr state approved (conditionally) or registration
+      supportedEntityFlags.includes(business.nameRequest.legalType) // Feature flagged Nr types
+    )
   }
 
-  /** Returns the Name value for the affiliation */
-  private name (item: Business): string {
-    return this.isNumberedIncorporationApplication(item) ? 'Numbered Benefit Company' : item.name
+  /** Returns the name of the affiliation. */
+  protected name (item: Business): string {
+    if (this.isNumberedIncorporationApplication(item)) {
+      const legalType: unknown = item.corpType.legalType
+      // *** TODO: remove fallback once namex API returns legal type
+      return GetCorpNumberedDescription(legalType as CorpTypeCd) || 'Numbered Company'
```

Comment:
Need to remove fallback value. (Or maybe not? What's better if something goes wrong in the data -- to see Numbered Company or to see nothing?)